src/sds.c

```diff
@@ -469,10 +469,11 @@ sds sdscatfmt(sds s, char const *fmt, ...) {
             s = sdsMakeRoomFor(s,1);
             sh = (void*) (s-(sizeof(struct sdshdr)));
         }
-
         switch(*f) {
         case '%':
             next = *(f+1);
+			if(next == 0)
+				break;
```

Comment:
```suggestion
			if(next == '\0') break;
```
Code style and clarity.

Reply:
Tab needs to be changed to space , indentation also needs to be optimized.

Reply:
@sundb thanks, fixed in suggestion above.